### PR TITLE
Code Review Updates

### PR DESCRIPTION
**Pull request by Falcons1231**
Project Description:
This is an app that allows users to search for recycling centers nearby and more specifically recycling centers that recycle specific items such as phones or batteries.

What I learned from the code review:
- I learned that OutlinedTextField doesn't have a background color property, but you can use TextField with modifiers such as shape, boarder, color to achieve a more customized TextField. 
- I learned that a Kotlin Best practice is to use a trailing comma at the end of a series of elements.

The project was able to build and was available for my review when I made the request to be added as a contributor.

### CODE REVIEW

File: app/src/main/java/com/findmyrecycling/MainActivity.kt [hunk 1]
```diff
@@ -35,19 +40,29 @@ fun RecycleSearch(name: String) {
     var location by remember{ mutableStateOf("")}
 
     Column {
-        OutlinedTextField(
+        TextField(
```
Comment (Falcons1231) [inline app/src/main/java/com/findmyrecycling/MainActivity.kt:43]: I found that using the TextField gave more flexibility with the desired modifications like background color

File: app/src/main/java/com/findmyrecycling/MainActivity.kt [hunk 2]
```diff
@@ -35,19 +40,29 @@ fun RecycleSearch(name: String) {
...
             value = recyclable,
             onValueChange = {recyclable = it},
             label = { Text(stringResource(R.string.recyclable)) },
-            modifier = Modifier.fillMaxWidth()
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
```
Comment (Falcons1231) [inline app/src/main/java/com/findmyrecycling/MainActivity.kt:49]: The TextFields were running up against the edges of the screen.  Adding padding to the start and end gave a better look and feel to the screen

File: app/src/main/java/com/findmyrecycling/ProfileScreen.kt [hunk 3]
```diff
@@ -38,22 +43,37 @@ fun ProfileOptions(name: String) {
...
             onValueChange = {profile = it},
             label = { Text(stringResource(R.string.profile)) },
-            modifier = Modifier.fillMaxWidth()
-        )
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
+                .border(width = 1.dp,
+                    color = Color.Black,
+                    shape = RoundedCornerShape(4.dp)),
```
Comment (Falcons1231) [inline app/src/main/java/com/findmyrecycling/ProfileScreen.kt:51]: Used trailing commas as recommended in Kotlin coding conventions here:  https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

File: app/src/main/java/com/findmyrecycling/ProfileScreen.kt [hunk 4]
```diff
@@ -38,22 +43,37 @@ fun ProfileOptions(name: String) {
...
             value = mySavedLocations,
             onValueChange = { mySavedLocations = it },
             label = { Text(stringResource(R.string.mySavedLocations)) },
-            modifier = Modifier.fillMaxWidth()
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
+                .border(width = 1.dp,
+                    color = Color.Black,
+                    shape = RoundedCornerShape(4.dp)),
```
Comment (Falcons1231) [inline app/src/main/java/com/findmyrecycling/ProfileScreen.kt:64]: Used trailing commas as recommended in Kotlin coding conventions here:  https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

File: app/src/main/java/com/findmyrecycling/ProfileScreen.kt [hunk 5]
```diff
@@ -38,22 +43,37 @@ fun ProfileOptions(name: String) {
...
             value = mySavedSearches,
             onValueChange = { mySavedSearches = it },
             label = { Text(stringResource(R.string.mySavedSearches)) },
-            modifier = Modifier.fillMaxWidth()
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
+                .border(width = 1.dp,
+                    color = Color.Black,
+                    shape = RoundedCornerShape(4.dp)),
```
Comment (Falcons1231) [inline app/src/main/java/com/findmyrecycling/ProfileScreen.kt:76]: Used trailing commas as recommended in Kotlin coding conventions here:  https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

File: app/src/main/java/com/findmyrecycling/ui/theme/Color.kt [hunk 6]
```diff
@@ -6,4 +6,5 @@ val Purple200 = Color(0xFFBB86FC)
 val Purple500 = Color(0xFF6200EE)
 val Purple700 = Color(0xFF3700B3)
 val Teal200 = Color(0xFF03DAC5)
-val RecyclingGreen = Color(0xFF009900)
\ No newline at end of file
+val RecyclingGreen = Color(0xFF009900)
+val RecyclingGray = Color(0xFFcccbc8)
```
Comment (Falcons1231) [inline app/src/main/java/com/findmyrecycling/ui/theme/Color.kt:10]: I noticed that the preview screen shot in the readMe file used gray background for your text fields so I added a Gray color to the theme.  0xFFcccbc8

File: app/src/main/java/com/findmyrecycling/MainActivity.kt (hunk 1 above)
Comment (discospiff) [reply app/src/main/java/com/findmyrecycling/MainActivity.kt:43]: Ahhh, are you sure?  It looks like the same function list to me.

https://developer.android.com/reference/kotlin/androidx/compose/material/package-summary#TextField(androidx.compose.ui.text.input.TextFieldValue,kotlin.Function1,androidx.compose.ui.Modifier,kotlin.Boolean,kotlin.Boolean,androidx.compose.ui.text.TextStyle,kotlin.Function0,kotlin.Function0,kotlin.Function0,kotlin.Function0,kotlin.Boolean,androidx.compose.ui.text.input.VisualTransformation,androidx.compose.foundation.text.KeyboardOptions,androidx.compose.foundation.text.KeyboardActions,kotlin.Boolean,kotlin.Int,androidx.compose.foundation.interaction.MutableInteractionSource,androidx.compose.ui.graphics.Shape,androidx.compose.material.TextFieldColors)

File: app/src/main/java/com/findmyrecycling/ui/theme/Color.kt (hunk 6 above)
Comment (discospiff) [reply app/src/main/java/com/findmyrecycling/ui/theme/Color.kt:10]: Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

**Comment by discospiff** [comment]
From this review, I'd consider harvesting the padding addition, if that fits within the team's UX design.

**Pull request merged**